Vagrantfile

```diff
@@ -3,6 +3,12 @@
 
 require 'yaml'
 
+unless File.exist?("settings.yml")
+  raise Vagrant::Errors::VagrantError.new, 
+    "'settings.yml' file not found.\n\n" \
+    "Please define your 'settings.yml'. See 'settings.sample.yml' for an example."
+end
```

Comment:
I believe you have the same semantic with the following syntax:

```
raise "'settings.yml' file not found.\n\n" \
         "Please define your 'settings.yml'. See 'settings.sample.yml' for an example."
unless File.exist?("settings.yml")
```
Can you check it?